src/app/views/preferences/customisation/customisation.component.html

```diff
@@ -60,5 +60,5 @@
       <label for="custom-input"></label>
     </div>
   </div>
-  <small class="goals__info">Some more explanation. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</small>
+  <small class="goals__info">This CO<sub>2</sub> Challenge has influence on the diagrams in your personal balance sheet. It is the value you dont want to overstep counting the last 30 days. The average value is sourced from the Federal Ministry of Environment (Germany).</small>
```

Comment:
I always used the term "charts" rather than "diagrams". Maybe replace for consistency?
Replace "dont" with "don't"

Reply:
replaced both